stsynphot/spectrum.py

```diff
@@ -598,7 +598,7 @@ def load_vega(vegafile=None, **kwargs):
 
     Returns
     -------
-    sp : `synphot.spectrum.SourceSpectrum` or `None`
+    sp : synphot.spectrum.SourceSpectrum, None
```

Comment:
Why? Now it does not link back to the SourceSpectrum API.

https://stsynphot--164.org.readthedocs.build/en/164/api/stsynphot.spectrum.load_vega.html

Reply:
Good point, I didn't want it to error but it looks like it works fine with the link